README.md

```diff
@@ -88,16 +88,15 @@ This distribution currently supports:
 
 ## Supported Kubernetes distributions
 
-This helm chart is tested and works with default configurations on the following
-Kubernetes distributions:
+The Helm chart works with default configurations of the main Kubernetes distributions. Use actively supported versions:
```

Comment:
>Use actively supported versions

Can we add some clarity around this like "support is only provided for <insert provider names> maintained versions per their ~license agreements" and "support for expired versions is best effort and backward incompatible feature adoption cannot be circumvented" (cc @aunshc)

Reply:
I'll propose we iterate on updating this in future PRs and in customer docs and move forward with this current version for this PR.